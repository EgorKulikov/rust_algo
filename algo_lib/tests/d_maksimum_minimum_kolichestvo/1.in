4
3
5 4 5
3
4 5 4
10
3 3 3 3 4 1 2 3 5 4
10
17 89 92 42 29 41 92 14 70 45
