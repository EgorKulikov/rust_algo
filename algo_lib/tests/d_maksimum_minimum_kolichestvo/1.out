12
11
12
186
